roles/reproducer/tasks/main.yml

```diff
@@ -134,3 +134,19 @@
       when:
         - cifmw_architecture_scenario is defined
       ansible.builtin.include_tasks: configure_architecture.yml
+
+    - name: Prepare ci-like EDPM deploy
+      ansible.builtin.copy:
+        dest: "/home/zuul/deploy-edpm.sh"
+        mode: "0755"
+        owner: "zuul"
+        group: "zuul"
+        content: |-
+          #!/bin/bash
```

Comment:
Shouldn't this be `/usr/bin/bash`?

Reply:
a quick `git grep /bin/bash` in the repository shows a vast majority of `/bin/bash` with maybe 3 `/usr/bin/bash` - both are valid.

Reply:
I was also under the impression that `/bin/bash` is valid until I hit an issue with coreOS. Since this is on the controller, I don't see a reason why we cannot use `/bin/bash`